lib/install/config/shared.js

```diff
@@ -1,10 +1,16 @@
 // Note: You must restart bin/webpack-watcher for changes to take effect
 
 var path = require('path')
+var process = require('process')
 var glob = require('glob')
 var extname = require('path-complete-extname')
+var distPath = process.env.WEBPACK_DIST_PATH
```

Comment:
Same. used to provide packs path from rails config into webpack configs.
For example i'm use it like this in production:
```js
new CleanWebpackPlugin([sharedConfig().dist], {
      root: path.resolve('..', 'public'),
      exclude: ['.keep']
  })
```